src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -132,7 +132,7 @@ public class C
                 },
                 comp =>
                 {
-                    var c = comp.GetTypeByMetadataName("C");
+                    var c = ((Compilation)comp).GetTypeByMetadataName("C");
```

Comment:
For the tests in this file, we should update them to test both the old API and the new public API in this change.

Reply:
> For the tests in this file, we should update them to test both the old API and the new public API in this change.

I don't think this absolutely have to be done in this PR. I will capture this feedback in an issue.

---
In reply to: [339256743](https://github.com/dotnet/roslyn/pull/39498#discussion_r339256743) [](ancestors = 339256743)

Reply:
Unless, this involves modifying small number of helpers 

---
In reply to: [339257849](https://github.com/dotnet/roslyn/pull/39498#discussion_r339257849) [](ancestors = 339257849,339256743)

Reply:
The concern I have is that the tests in this file are for the public nullable API. Unless I missed something, I didn't see any new tests added to test the new API added to `ITypeSymbol`, and the appropriate way to add coverage is to modify the tests in this file to add new coverage. For the tests formatted like this, it should be simple to modify the helper, but the other tests shouldn't be very complicated either.

---
In reply to: [339258244](https://github.com/dotnet/roslyn/pull/39498#discussion_r339258244) [](ancestors = 339258244,339257849,339256743)